CHANGELOG.md

```diff
@@ -53,7 +53,7 @@ Extends support of the SDK to OneView REST API version 2000 (OneView v5.40).
 #### Bug fixes & Enhancements
 - [#81](https://github.com/HewlettPackard/oneview-python/issues/81) EthernetNetworks Update does not work.
 - [#91](https://github.com/HewlettPackard/oneview-python/issues/91) Change Password Sample?
-
+- [#84](https://github.com/HewlettPackard/oneview-python/issues/84) enclosure_groups.create() Does not take valid data and returns JSON error.
```

Comment:
Since 5.4.0 is already released and fix is not available in that release, add this issue under 5.5.0 release.
Also update the issue 84 to 103

Reply:
Done